src/drive/styles/table.styl

```diff
@@ -90,6 +90,11 @@
 .fil-content-header-sortabledesc
 .fil-content-header-sortableasc
     cursor pointer
+.fil-content-dropzone
+    border 1px solid grey
+
+.fil-content-dropzone-active
+    background-color #F1F7FF
```

Comment:
Can we use the named color please? [It's called `zircon`](https://docs.cozy.io/cozy-ui/styleguide/section-settings.html#kssref-settings-colors-blue).